sfepy/discrete/functions.py

```diff
@@ -42,6 +43,52 @@ def set_function(self, function, is_constant=False):
     def set_extra_args(self, **extra_args):
         self.extra_args = extra_args
 
+def make_sfepy_function(fun_or_name=None):
+    """
+    Convenience decorator to quickly create
+    sfepy.discrete.functions.Function objects
+
+    Has two modes of use either without parameter:
```

Comment:
Use :: instead of : to make the code block rendered properly:
```
It has two modes of use, either with or without a parameter::
```

Reply:
Fixed, also added :: to the later comment line or with name:: to get two block.